libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -35,6 +35,11 @@
 
 static void * kObservingKey = &kObservingKey;
 
+@interface CSFNetwork() <SFUserAccountManagerDelegate> {
```

Comment:
I'm presuming you just wanted the `SFUserAccountManagerDelegate` part, right?  You don't want the `CSFNetwork` private interface here.  You can piggy back the delegate declaration in `CSFSalesforceAction_Internal.h`.